.github/workflows/setup/action.yml

```diff
@@ -0,0 +1,36 @@
+name: setup
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v2
+    - uses: davidB/rust-cargo-make@v1
+    - uses: actions-rs/install@v0.1
+      with:
+        crate: cargo-insta
+        version: latest
+        use-tool-cache: true
+    - name: Gen bindings
+      run: cargo make init-bindings
+      shell: bash
+
+    - name: Prepare frontend
+      run: npm install -g graco && cd frontend && graco prepare
+      shell: bash
+
+    - name: Install aquascope
+      run: cargo install --path crates/aquascope_front --debug
+      shell: bash
+
+    - name: Install mdbook-aquascope
+      run: cargo install --path crates/mdbook-aquascope --debug
+      shell: bash
+
+    - name: Setup mdBook
+      run: cargo install mdbook
```

Comment:
Let's download the prebuilt binary to reduce the likely-egregious CI times. This is how TRPL does it:

```yaml
    - name: Install mdbook
      run: |
        mkdir bin
        curl -sSL https://github.com/rust-lang/mdBook/releases/download/v0.4.14/mdbook-v0.4.14-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=bin
        echo "$(pwd)/bin" >> ${GITHUB_PATH}
```